cosmos/operators/watcher.py

```diff
@@ -379,15 +383,15 @@ class DbtSnapshotWatcherOperator(DbtSnapshotMixin, DbtConsumerWatcherSensor):  #
     Watches for the progress of dbt snapshot execution, run by the producer task (DbtProducerWatcherOperator).
     """
 
-    template_fields: tuple[str] = DbtConsumerWatcherSensor.template_fields
+    template_fields: tuple[str, str] = DbtConsumerWatcherSensor.template_fields
 
 
 class DbtSourceWatcherOperator(DbtSourceLocalOperator):
     """
     Executes a dbt source freshness command, synchronously, as ExecutionMode.LOCAL.
     """
 
-    template_fields: Sequence[str] = DbtSourceLocalOperator.template_fields
+    template_fields: tuple[str, str] = DbtConsumerWatcherSensor.template_fields + DbtRunMixin.template_fields  # type: ignore[operator]
```

Comment:
```suggestion
    template_fields: tuple[str, str] = DbtConsumerWatcherSensor.template_fields
```

Why DbtRunMixin? It does not extend the Mixin either.